src/components/form/partitions/DateRange.js

```diff
@@ -30,6 +30,10 @@ class DateTime extends PureComponent {
     });
   };
 
+  handleDateChange = (id, oldDate) => value => {
+    this.onChange(id)(oldDate ? value : addHours(value, 9));
```

Comment:
I do not immediately understand why 9 hours should be added, could it be clarified with e.g. a comment?

Reply:
They wanted to set some other starting position for time than 00:00 by default. So I added 9 hours when selecting new date. This is little hacky fealing solution but I didn't find any helpful props from react-datepicker API

Reply:
Ah, makes sense